gbpservice/nfp/configurator/lib/schema.py

```diff
@@ -1,17 +1,71 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Defines schema of the resource.This file contains all the generic config
+   resources. Schema name defined here must be same as resource name in the
+   request_data.
+   Format of request data for network function device configuration API:
+
+   request_data {
+        info {
+            version: <v1/v2/v3>
+        }
+        config [
+            {
+                'resource': <healthmonitor/routes/interfaces>,
+                'kwargs': <resource parameters>
+            },
+            {
+            'resource': <healthmonitor/routes/interfaces>,
+            'kwargs': <resource parameters>
+            }, ...
+        ]
+    }
+
+"""
+
+
 skip_kwargs_validation_for = ['firewall', 'vpn', 'loadbalancer']
 
+
+""" Request data schema.
+"""
+
 request_data = {'info': {},
                 'config': []
                 }
 
-request_data_info = {'version': "",
-                     'service_type': ""
```

Comment:
Provided a comment on earlier PR on configurator to add the request_data_info also for the validation. Can we address this.